ptype/plotting.py

```diff
@@ -0,0 +1,132 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from sklearn.metrics import confusion_matrix
+
+def ptype_hist(df, col, dataset, bins=None, filename=None):
+    """
+    Function to plot a histogram of a specified variable when
+    the percent of each ptype is greater than 0.
+    """
+    ra = df[col][df['ra_percent'] > 0]
+    sn = df[col][df['sn_percent'] > 0]
+    pl = df[col][df['pl_percent'] > 0]
+    fzra = df[col][df['fzra_percent'] > 0]
+    
+    fig, ax = plt.subplots(2, 2, figsize=(12,10))
+    
+    if bins is None:
+        ax[0,0].hist(ra, density=True)
```

Comment:
It would be more compact to replace this with a for loop that enumerates over the different axes. For example
```python
for p, ptype in enumerate([ra, sn, pl, fzra]):
   ax.ravel()[p].hist(ptype, density=True)
   ...
```